src/bin/scribble.ts

```diff
@@ -629,11 +629,16 @@ if ("version" in options) {
         }
     }
 
+    const instrumentationMarker =
+        "/// This source is auto-generated by Scribble and shouldn't be edited directly.\n" +
```

Comment:
nit: source -> file would be a little cleaner

Reply:
Will fix

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/127/commits/b76dea9190f54c5098a65840961143fdb78b17d8